server/src/main/java/com/linecorp/centraldogma/server/CentralDogma.java

```diff
@@ -510,6 +512,9 @@ private Server startServer(ProjectManager pm, CommandExecutor executor,
         sb.clientAddressSources(cfg.clientAddressSourceList());
         sb.clientAddressTrustedProxyFilter(cfg.trustedProxyAddressPredicate());
 
+        // Config cors policy
```

Comment:
This comment is a kind of repetitive. The method name `configCors` itsels tells it all.
```suggestion
```